lib/util/mongo.js

```diff
@@ -169,9 +169,18 @@ function getPrimary(params, cb) {
   });
 }
 
+function hasUserSpaceDb() {
+  if (process.env.MONGODB_USERDB_NAMESPACE) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
I guess just checking the existence of the env var is OK for this now since this is only ever going to be run on OpenShift.

Reply:
I think this is fine, maybe we could check if its non empty too though on second thoughts?

Reply:
It will validate to false as well if it is an empty string